src/main/java/core/basesyntax/models/Penguin.java

```diff
@@ -1,10 +1,11 @@
 package core.basesyntax.models;
 
+import core.basesyntax.interfaces.FlyAbility;
 import core.basesyntax.interfaces.SwimAbility;
 
-public class Penguin extends Bird implements SwimAbility {
+public class Penguin extends Bird implements SwimAbility, FlyAbility {
```

Comment:
сорі, помилково дописала FlyAbility (пінгвін же не може літати)
```suggestion
public class Penguin extends Bird implements SwimAbility {
```